kotlin/src/main/com/looker/rtl/Transport.kt

```diff
@@ -53,6 +49,16 @@ import javax.net.ssl.HostnameVerifier
 import javax.net.ssl.SSLContext
 import javax.net.ssl.SSLSocketFactory
 import javax.net.ssl.X509TrustManager
+import kotlin.collections.Map
+import kotlin.collections.component1
```

Comment:
Seems like a number of these were added, but I don't see them getting used here.  Do we have a linter set up that can help flag and remove unused imports?

Reply:
Better Kotlin linting it a todo for this repo/project

Reply:
Interestingly, IntelliiJ isn't complaining about any unused imports in that file

Reply:
Also, those got added when I told IntelliJ to go ahead and optimize the imports it was suggesting